HEBO/hebo/optimizers/bo.py

```diff
@@ -46,17 +51,31 @@ def suggest(self, n_suggestions = 1, fix_input = None):
                     sample[k] = v
             return sample
         else:
-            X, Xe     = self.space.transform(self.X)
-            y         = torch.FloatTensor(self.y)
-            num_uniqs = None if Xe.shape[1] == 0 else [len(self.space.paras[name].categories) for name in self.space.enum_names]
-            model     = get_model(self.model_name, X.shape[1], Xe.shape[1], y.shape[1], num_uniqs = num_uniqs, warp = False)
+            X, Xe = self.space.transform(self.X)
+            y = torch.FloatTensor(self.y)
+            num_uniqs = (

```

Comment:
would need to make this cleaner

Reply:
what do you mean? I formatted with black, I can revert this change if you like?